File: library/php/composer/{{cookiecutter.project_slug}}/src/Hello.php
<?php

declare(strict_types=1);

namespace {{ cookiecutter.php_namespace }};

/**
 * Dummy public class. Replace with the real API.
 */
final class Hello
{
    public static function greet(string $name = 'world'): string
    {
        return sprintf('hello, %s', $name);
    }
}

File: project/library/java/maven/{{cookiecutter.project_slug}}/src/main/java/{{cookiecutter.java_package_dir}}/Hello.java
package {{ cookiecutter.java_package }};

/** Dummy public class. Replace with the real API. */
public final class Hello {
    private Hello() {
    }

    public static String greet(String name) {
        return "hello, " + (name == null || name.isEmpty() ? "world" : name);
    }
}
